src/PowerShellEditorServices.Engine/Services/TextDocument/Handlers/DefinitionHandler.cs

```diff
@@ -0,0 +1,108 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.PowerShell.EditorServices;
+using Microsoft.PowerShell.EditorServices.Symbols;
+using OmniSharp.Extensions.LanguageServer.Protocol.Client.Capabilities;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+using OmniSharp.Extensions.LanguageServer.Protocol.Server;
+using PowerShellEditorServices.Engine.Utility;
+
+namespace PowerShellEditorServices.Engine.Services.Handlers
+{
+    public class DefinitionHandler : IDefinitionHandler
+    {
+        private readonly DocumentSelector _documentSelector = new DocumentSelector(
+            new DocumentFilter()
+            {
+                Language = "powershell"
```

Comment:
No `ps*1` bit?

Reply:
Patrick helped me realize in one of my last PRs that `ps*1` would only apply to real files, not untitled files. This is the right way to cover untitled files.